commons-http/src/main/scala/com/avsystem/commons/http/jetty/rest/RestClient.scala

```diff
@@ -1,5 +1,5 @@
 package com.avsystem.commons
-package jetty.rest
+package http.jetty.rest
```

Comment:
I would prefer if we left the packages as they were now. Let's leave project restructuring for later.